test/gpu/octree/test_radius_search.cpp

```diff
@@ -103,24 +103,34 @@ TEST(PCL_OctreeGPU, batchRadiusSearch)
         host_search1[i].reserve(max_answers);
         host_search2[i].reserve(max_answers);
     }    
-    
+
+    pcl::gpu::Octree::ResultSqrDists result_sqr_distances1;
+    pcl::gpu::Octree::ResultSqrDists result_sqr_distances2;
+    pcl::gpu::Octree::ResultSqrDists result_sqr_distances3;
+
     //search GPU shared
-    octree_device.radiusSearch(queries_device, data.shared_radius, max_answers, result_device1);
+    octree_device.radiusSearch(queries_device, data.shared_radius, max_answers, result_device1, result_sqr_distances1);
 
     //search GPU individual
-    octree_device.radiusSearch(queries_device,    radiuses_device, max_answers, result_device2);
+    octree_device.radiusSearch(queries_device,    radiuses_device, max_answers, result_device2, result_sqr_distances2);
 
     //search GPU shared with indices
     pcl::gpu::Octree::Indices indices;
     indices.upload(data.indices);
-    octree_device.radiusSearch(queries_device, indices, data.shared_radius, max_answers, result_device3);
+    octree_device.radiusSearch(queries_device, indices, data.shared_radius, max_answers, result_device3, result_sqr_distances3);
 
     //search CPU
     octree_device.internalDownload();
+
+    std::vector< std::vector<float> > host_sqr_distances1(data.tests_num);
+    std::vector< std::vector<float> > host_sqr_distances2(data.tests_num);
```

Comment:
Weird spacing. C++14 no longer complains about double "bigger than"  in templates.